turbofan/engine_validation.py

```diff
@@ -357,50 +340,26 @@ def setup(self, res7, cooling, N, state, eng, Nfleet=0, BLI=False, goption=1):
                     self.fanmap['\\bar{m}_{fan_{D}}'] <= 1.7 * self.sizing['\\alpha_{OD}'] * self.sizing['m_{coreD}']* ((250.0/288)**.5)/(50/101.325),
                     ]
 
-        if res7 == 0:
-            res7list = [
-                #residual 7
-                #option #1, constrain the engine's thrust
-                self.engineP['F'] == Fspec,
-                ]
-            if cooling and self.Ttmax:
-                Tt41max = Variable('T_{t_{4.1_{max}}}', 'K', 'Max turbine inlet temperature')
-                res7list.extend([
-                    self.engineP['T_{t_{4.1}}']  <= Tt41max,
-                    ])
-            elif self.Ttmax:
-                Tt4max = Variable('T_{t_{4_{max}}}', 'K', 'Max turbine inlet temperature')
-                res7list.extend([
-                    self.engineP['T_{t_4}'] <= Tt4max,
-                    ])
-        if res7 == 1:
-            if cooling:
-                res7list = [
-                    #residual 7
-                    #option #2 constrain the burner exit temperature
-                    self.engineP['T_{t_{4.1}}'] == Tt4spec,  #B.265
-                    ]
-            else:
-                res7list = [
-                    #residual 7
-                    #option #2 constrain the burner exit temperature
-                    self.engineP['T_{t_4}'] == Tt4spec,  #B.265
-                    ]
```

Comment:
are we losing functionality by getting rid of the `res7` option? Which option (1 or 0) does this new implementation correspond to?

Reply:
It's not a loss of functionality. Now we can actually specify Tt4.1max instead of setting it directly as a parameter. I just no longer see a use case for setting Tt4 directly. This will also make turbofan more user friendly.